docs/pages/more/expo-cli.mdx

```diff
@@ -340,6 +341,24 @@ This command will be disabled if your project is configured to use `metro` for b
 
 Native source code must be generated before a native app can compile. Expo CLI provides a unique and powerful system called _prebuild_, that generates the native code for your project. To learn more, read the [Expo Prebuild docs](/workflow/prebuild/).
 
+## Lint
+
+<Terminal cmd={['$ npx expo lint']} />
+
+Linting helps enforce best practices and ensure your code is consistent. The `npx expo lint` command will setup ESLint with Expo-specific settings and run the `npx eslint` command with options that are optimized for the Expo framework. You can fix all automatically fixable issues by running `npx expo lint --fix`.
+
+Running `npx expo lint` targets all files in the **/src**, **/app**, and **/components** directories by default. You can pass custom files or directories to lint by passing them as arguments, e.g. `npx expo lint ./utils constants.ts`.
```

Comment:
```suggestion
Running `npx expo lint` targets all files in the **/src**, **/app**, and **/components** directories by default. You can also pass custom files or directories to lint by passing them as arguments. For example, `npx expo lint ./utils constants.ts`.
```

The second sentence is an option feature description. Let's emphasize it that way.